--java.lang.AssertionError: ALTER TABLE t0 COMPRESSION 'NONE', FORCE, STATS_PERSISTENT 1, ROW_FORMAT FIXED, STATS_AUTO_RECALC 0, ALGORITHM INSTANT, PACK_KEYS 0, DELAY_KEY_WRITE 1, CHECKSUM 1;
--	at sqlancer.common.query.SQLQueryAdapter.checkException(SQLQueryAdapter.java:111)
--	at sqlancer.common.query.SQLQueryAdapter.execute(SQLQueryAdapter.java:93)
--	at sqlancer.Main$QueryManager.execute(Main.java:265)
--	at sqlancer.GlobalState.executeStatement(GlobalState.java:108)
--	at sqlancer.StatementExecutor.executeStatements(StatementExecutor.java:70)
--	at sqlancer.mysql.MySQLProvider.generateDatabase(MySQLProvider.java:152)
--	at sqlancer.mysql.MySQLProvider.generateDatabase(MySQLProvider.java:1)
--	at sqlancer.ProviderAdapter.generateAndTestDatabase(ProviderAdapter.java:52)
--	at sqlancer.Main$DBMSExecutor.run(Main.java:364)
--	at sqlancer.Main$2.run(Main.java:559)
--	at sqlancer.Main$2.runThread(Main.java:541)
--	at sqlancer.Main$2.run(Main.java:532)
--	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
--	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
--	at java.base/java.lang.Thread.run(Thread.java:829)
--Caused by: java.sql.SQLException: Unknown ALGORITHM 'INSTANT'
--	at com.mysql.cj.jdbc.exceptions.SQLError.createSQLException(SQLError.java:129)
--	at com.mysql.cj.jdbc.exceptions.SQLExceptionsMapping.translateException(SQLExceptionsMapping.java:122)
--	at com.mysql.cj.jdbc.StatementImpl.executeInternal(StatementImpl.java:763)
--	at com.mysql.cj.jdbc.StatementImpl.execute(StatementImpl.java:648)
--	at sqlancer.common.query.SQLQueryAdapter.execute(SQLQueryAdapter.java:87)
--	... 13 more
---- Time: 2023/06/03 17:49:25
-- Database: database46
-- Database version: 8.0.11
-- seed value: 1685807359593
DROP DATABASE IF EXISTS database46;
CREATE DATABASE database46;
USE database46;
CREATE TABLE t0(c0 TINYINT  COLUMN_FORMAT DYNAMIC STORAGE MEMORY UNIQUE KEY ) ;
INSERT IGNORE INTO t0(c0) VALUES(']');
